.github/workflows/bot-assign-on-comment.yml

```
@@ -0,0 +1,38 @@
+name: GFI Assign on /assign
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)